Sources/Build/misc.swift

```diff
@@ -77,9 +78,9 @@ extension ClangModule {
         
         let umbrellaHeader = Path.join(moduleHeaderDir, "\(name).h")
         if umbrellaHeader.isFile {
-            try createModuleMap(.HeaderFile)
+            try createModuleMap(inDir: wd, type: .HeaderFile)
         } else {
-            try createModuleMap(.ModuleNameDir)
+            try createModuleMap(inDir: wd, type: .ModuleNameDir)
```

Comment:
As discussed, modulemap allows `umbrella "dir"` so not listing headers explicitly.